bbp/default.nix

```diff
@@ -415,6 +415,12 @@ let
             numpy = pythonPackages.numpy;
         };
 
+        flatindexer-py3 = callPackage ./hpc/FLATIndexer {
+            mpiRuntime = bbp-mpi;
+            boost = callPackage ../std-nixpkgs/pkgs/development/libraries/boost/1.58.nix { python=python3; };
```

Comment:
same for boost, a simple "override { python = python3} " should make the trick. I'm surprise it is not done soemwhere already.